Test.C

```diff
@@ -146,30 +147,49 @@ int main(){
     // rounds with -1 do not upgrade attack
     Round rounds[NUM_ROUNDS] = {
             {1, 0},
```

Comment:
```suggestion
            {1, 1},
```

For consistency, 1 is attack will be generated, -1 it will not